src/test/java/wooteco/subway/dao/StationDaoTest.java

```diff
@@ -0,0 +1,63 @@
+package wooteco.subway.dao;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import wooteco.subway.domain.Station;
+
+@JdbcTest
+class StationDaoTest {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    private StationDaoImpl stationDao;
+
+    @BeforeEach
+    void beforeEach() {
+        stationDao = new StationDaoImpl(jdbcTemplate);
+    }
+
+    @Test
+    @DisplayName("지하철 역을 저장할 수 있다.")
+    void insert() {
+        String name = "강남역";
+        Station station = stationDao.insert(new Station(name));
+
+        assertThat(station.getName()).isEqualTo(name);
+    }
+
+    @Test
+    @DisplayName("이미 존재하는 이름의 지하철 역인지 확인한다.")
+    void existByName() {
+        Station station = stationDao.insert(new Station("강남역"));
+        Boolean actual = stationDao.existByName(new Station("강남역"));
+
+        assertThat(actual).isEqualTo(true);
```

Comment:
```suggestion
        assertThat(actual).isTrue();
```
이런식으로도 사용할 수 있습니다 :)